src/hotspot/os/windows/os_windows.cpp

```diff
@@ -718,6 +718,7 @@ bool os::create_thread(Thread* thread, ThreadType thr_type,
     case os::gc_thread:
     case os::asynclog_thread:
     case os::watcher_thread:
+    case os::os_thread:
```

Comment:
How did you determine this is the correct behaviour for this type of thread? This will result in a change in behaviour.